dts/riscv/neorv32.dtsi

```diff
@@ -30,6 +30,7 @@
 				#interrupt-cells = <1>;
 
 				firq: firq {
+					#address-cells = <2>;
```

Comment:
Why is it set to `2`?

Reply:
Oh, good catch. I actually got this wrong and it should be 1 to be correct.

But I wonder if this entry is actually misformed, why is the interrupt-map in a subnode instead of in the main interrupt controller node in the first place?

Reply:
Hm, I wonder if the interrupt-map should be in the soc node instead in this case. But I don't have hardware to verify with if I change it.